// caldb/bk/index.js
var Database = require('better-sqlite3');
var db = new Database(__dirname+'/db/eve.db');

// === eve ===
//
exports.mailEve=function(email){
var stm=db.prepare('select * from eve where email=?');
var row=stm.all(email);
return row;
};

exports.mon1=function(email){
var stm=db.prepare('select * from mon'+1+' where email=?');
var row=stm.all(email);
return row;
};


exports.mon2=function(email){
var stm=db.prepare('select * from mon2 where email=?');
var row=stm.all(email);
return row;
};

exports.mon3=function(email){
var stm=db.prepare('select * from mon3 where email=?');
var row=stm.all(email);
return row;
};

exports.mon4=function(email){
var stm=db.prepare('select * from mon4 where email=?');
var row=stm.all(email);
return row;
};

exports.mon5=function(email){
var stm=db.prepare('select * from mon5 where email=?');
var row=stm.all(email);
return row;
};

exports.mon6=function(email){
var stm=db.prepare('select * from mon6 where email=?');
var row=stm.all(email);
return row;
};

exports.namEve=function(arg2){
var stm=db.prepare('select * from eve where eve=?');
try{var row=stm.all(arg2);
}catch(err){console.log(err);}
return row;
};

exports.allEve=function(){
var stm=db.prepare('select * from eve ');
var row=stm.all();
return row;
};


// === eve run ===
exports.updEve=function(eve,mail){
var stm=db.prepare('update eve set eve=? where email=?');
try{stm.run(eve,mail);
}catch(err){console.log(err);}
};

exports.insEve=function(eve,email,date,des){
var stm=db.prepare('insert into eve values(?,?,?,?) ');
stm.run(eve,email,date,des);
};

exports.delEve=function(){
var stm=db.prepare('delete from eve ');
try{stm.run();
}catch(err){var err=err;console.log(err);}
};
