// predb/index.js
var Database = require('better-sqlite3');
var db = new Database(__dirname+'/db/rob.db');

// === usr ===
// ins
exports.inUsr=function(name,mail,pss,sid){
var stm=db.prepare('insert into usr values(?,?,?,?)')
try{var row=stm.run(name,mail,pss,sid)}
catch(err){console.log(err);}
};

//upd
exports.upSid=function(sid){
var stm=db.prepare('update usr set sid=? where mail=? )')
try{stm.run(sid)
}catch(err){console.log(err);}
};

exports.selUsr=function(mail,pss){
var stm=db.prepare('select * from usr where mail=? and pss=?');
try{var row=stm.all(mail.pss);
}catch(err){console.log(err);}
return row;
};
// === pre ===
exports.selPre=function(arg2){
var stm=db.prepare('select * from pre where type=?');
try{var row=stm.all(arg2);
}catch(err){console.log(err);}
return row;
};

exports.allPre=function(){
var stm=db.prepare('select *,rowid from pre' )
try{var row=stm.all();
}catch(err){console.log(err);}
return row;
};
// ins
exports.inPre=function(name,mail,item,unit,flg){
var stm=db.prepare('insert into pre values(?,?,?,?,?)')
try{var row=stm.run(name,mail,item,unit,flg)}
catch(err){console.log(err);}
};

exports.upFlg=function(flg,id){
var stm=db.prepare('update pre set flg=? where rowid=? ')
try{stm.run(flg,id)
}catch(err){console.log(err);}
};
// mer
exports.allMer=function(){
var stm=db.prepare('select *,rowid from mer' )
try{var row=stm.all();
}catch(err){console.log(err);}
return row;
};

exports.selMer=function(arg2){
var stm=db.prepare('select * from mer where name=?');
try{var row=stm.get(arg2);
}catch(err){console.log(err);}
return row;
};

exports.skuMer=function(arg2){
var stm=db.prepare('select * from mer where sku=?');
try{var row=stm.get(arg2);
}catch(err){console.log(err);}
return row;
};
