// dblo/index.js
var Database = require('better-sqlite3');
var db = new Database(__dirname+'/db/blo.db');

// === log ===
exports.typBlo=function(arg2){
var stm=db.prepare('select * from blo where type=?');
try{var row=stm.all(arg2);
}catch(err){console.log(err);}
return row;
};

exports.allBlo=function(){
var stm=db.prepare('select * from blo');
var row=stm.all();
return row;
};

exports.terBlo=function(ter){
var stm=db.prepare('select * from blo where term=?');
try{var row=stm.get(ter);
}catch(err){console.log(err);}
return row;
};

// ins
exports.inPos=function(tit,typ,ter,lin,bod,dat){
var stm=db.prepare('insert into blo values(?,?,?,?,?,?)')
try{var row=stm.run(tit,typ,ter,lin,bod,dat);
}catch(err){console.log(err);}
};

//upd
exports.upTyp=function(type,tit){
var stm=db.prepare('update blo set type=? where tit=? )')
try{stm.run(type,tit)
}catch(err){console.log(err);}
};

exports.upTer=function(typ,term){
var stm=db.prepare('update blo set term=? where tit=? )')
try{stm.run(typ,tit)
}catch(err){console.log(err);}
};
